src/common/UserAvatar.js

```diff
@@ -45,24 +55,25 @@ function UserAvatar(props: Props) {
   }
 
   return (
-    <View>
-      <Touchable onPress={onPress} style={style}>
-        <ImageBackground
-          style={style}
-          source={{
-            uri: avatarUrl.get(PixelRatio.getPixelSizeForLayoutSize(size)).toString(),
-            ...(avatarUrl instanceof FallbackAvatarURL
-              ? { headers: getAuthHeaders(auth) }
-              : undefined),
-          }}
-          resizeMode="cover"
-          /* ImageBackground seems to ignore `style.borderRadius`. */
-          borderRadius={borderRadius}
```

Comment:
In the old code, this translated to a `borderRadius={borderRadius}` on the `Image`. Was dropping that intentional?

Reply:
We apply the border radius via `style`, so doing it here is redundant. I probably should have done that as a separate commit, though.